tests/functional/modules/test_zos_job_output_func.py

```diff
@@ -47,7 +45,6 @@ def test_zos_job_output_invalid_job_id(ansible_zos_module):
     results = hosts.all.zos_job_output(job_id="INVALID")
     for result in results.contacted.values():
         assert result.get("changed") is False
-        assert result.get("jobs")[0].get("ret_code").get("msg_txt") == JOB_NOT_FOUND_MSG_TXT_ID
```

Comment:
At least could we keep checking that this is not None ? Or if it changed just verify that the stdout for error is there.

Reply:
This is still removed, could you restore the validation or provide a reason on why to remove it ?

Reply:
Marcel checked that stderr is not none